nixos/modules/services/web-apps/bookstack.nix

```diff
@@ -228,6 +276,30 @@ in {
       }
     ];
 
+    services.bookstack.config = lib.mapAttrs (_: lib.mkDefault) {
+      APP_KEY._secret = cfg.appKeyFile;
```

Comment:
While that specific scenario wouldn't work, since only `_secret` attributes are allowed, you're right that it's a bit odd to only make the values default. I'll get rid of the `mkDefault`.

Hm, yeah, that's a good point. I'll make it a submodule instead.

Reply:
Should be fixed now.

Reply:
yup, looks good :)